api/src/components/publication/controller.ts

```diff
@@ -67,6 +67,28 @@ export const get = async (
     }
 };
 
+export const getPubSeed = async (
```

Comment:
It's not clear what is this handler supposed to do by it's name.

Please fix eslint and prettier warnings.

<img width="1344" alt="image" src="https://user-images.githubusercontent.com/48569671/204018132-257df548-b528-49c1-b917-e6382b6b8776.png">